src/System.Management.Automation/utils/PSTelemetryMethods.cs

```diff
@@ -196,12 +196,12 @@ internal static void ReportModuleLoad(PSModuleInfo foundModule)
             System.Management.Automation.Host.TranscriptionData transcriptionData)
         {
             bool isConstrained = (iss != null) && (iss.DefaultCommandVisibility != SessionStateEntryVisibility.Public) && (iss.LanguageMode != PSLanguageMode.FullLanguage);
-            bool isTranscripting = (transcriptionData != null) && (transcriptionData.SystemTranscript != null);
+            bool isTranscribing = (transcriptionData != null) && (transcriptionData.SystemTranscript != null);
 
             TelemetryWrapper.TraceMessage("PSNewLocalSession", new
             {
                 Constrained = isConstrained,
-                Transcripting = isTranscripting
+                Transcribing = isTranscribing
```

Comment:
Not a native English speaker, so would need some help here. Is "Transcripting" a correct gerund form for "transcript"? Google translate doesn't seem to know it, but from my understanding "Transcribe" is slightly different from "Transcript". Also, this is a telemetry that we already using. @lzybkr what do you think about this change?